### DISCUSSION TITLE
MCapPts File Output criteria

### DISCUSSION THREAD

**Discussion by SONOFSWAAD** [devfinwiz/Stock_Screeners_Raw#1] [Q&A] [answered]
How do you determine that the following set of stocks are undervalued? How does your script arrive to that conclusion? And how should I use it for analysis?

**Comment by devfinwiz** [comment, marked as answer]
Actually, it is a lesser known strategy combo among new investors in the market. The script finds out stocks that have market cap less than their annual sales. These set of stocks can then later be used by investors to spot investment opportunities. For example: Tata Motors have a very less market cap as compared to its annual sales and we know how its EV business is picking up at a decent pace. Hence, this can be an investment opportunity which is hidden to the investors as the company is currently loss making and no other parameter like PE can be used to spot the stock as undervalued stock. General thumb rule: A company's market cap and its annual sales must not have a wide gap between them if the company is truly a good investment opportunity.
